index.html

```diff
@@ -115,6 +115,37 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
Please change this 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` comment to
`<!-- ________ YourName Contributor card START ________ -->`

Reply:
This comment helps others to identify your card in the HTML file.